edx_exams/apps/api/v1/views.py

```diff
@@ -1 +1,180 @@
-# Create your views here.
+"""
+V1 API Views
+"""
+import logging
+import uuid
+
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from edx_api_doc_tools import path_parameter, schema
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+
+from edx_exams.apps.api.permissions import StaffUserPermissions
+from edx_exams.apps.api.serializers import ExamSerializer
+from edx_exams.apps.core.exam_types import get_exam_type
+from edx_exams.apps.core.models import CourseExamConfiguration, Exam
+
+
+log = logging.getLogger(__name__)
+
+
+class ExamsView(APIView):
+    """
+    View to modify exams.
```

Comment:
I think the comments could be a bit clearer calling out this endpoint modifies all exams for a course.